src/main/java/lotto/LottoGenerator.java

```diff
@@ -0,0 +1,45 @@
+package lotto;
+
+import utils.DrawNumber;
+
+import java.util.List;
+
+public final class LottoGenerator {
+	private final static int LOTTO_NUMBER_COUNT = 6;
+	private final static int LOTTO_PRICE = 1000;
+
+	private final LottoNumbersFactory lottoNumbersFactory;
+	private final DrawNumber drawNumber;
+
+	public LottoGenerator(final LottoNumbersFactory lottoNumbersFactory, final DrawNumber drawNumber){
+		this.lottoNumbersFactory = lottoNumbersFactory;
+		this.drawNumber = drawNumber;
+	}
+
+	public int buy(final int price){
+		int lottoNumber = price / LOTTO_PRICE;
+		if (lottoNumber <= 0) {
+			return 0;
+		}
+
+		for(int i = 0; i < lottoNumber; ++i){
```

Comment:
이곳엔 `로또 번호 갯수`를 나타내는 다른 `네이밍`을 주는게 좋겠네요.
lottoNumber는 로또 번호 하나를 의미하는 것 같네요 :) 

```suggestion
		for (int i = 0; i < lottoNumber; ++i) {
```

Reply:
의도랑 전혀 상관없이 네이밍된 것 같습니다. d0fe336에서 수정했습니다.